docs/messaging.rst

```diff
@@ -184,7 +184,7 @@ which is the topic for IOPub subscribers, e.g. ``execute_result``, ``display_dat
     In most cases, the IOPub topics are irrelevant and completely ignored,
     because frontends just subscribe to all topics.
     The convention used in the IPython kernel is to use the msg_type as the topic,
-    and possibly extra information about the message, e.g. ``execute_result`` or ``stream.stdout``
+    and possibly extra information about the message, e.g. ``kernel.{u-u-i-d}.execute_result`` or ``stream.stdout``
```

Comment:
Ah, k. Looks like I'll need to dig a little bit deeper later. I've been ignoring identities in my current code at the moment.